packages/server/package.json

```diff
@@ -20,9 +20,12 @@
   "dependencies": {
     "@accounter/green-invoice-graphql": "^0.7.2",
     "@accounter/pcn874-generator": "^0.4.1",
+    "@apollo/subgraph": "2.9.3",
     "@envelop/generic-auth": "8.0.1",
     "@envelop/graphql-modules": "6.0.0",
+    "@graphql-hive/gateway": "1.7.6",
```

Comment:
I think we can remove the `hive` dependencies from this package right?
both `@graphql-hive/gateway` and `@graphql-hive/cli`?

Reply:
Done :)